src/deploy/BaseDeployer.js

```diff
@@ -97,7 +97,7 @@ class BaseDeployer {
         return;
       }
       this.log.info(`id: ${chalk.grey(id)}`);
-      if (ret.status === 404 && retry404) {
+      if ((ret.status === 404 || ret.status === 500) && retry404) {
```

Comment:
when omitting the route creation, deploying the action is sometimes too quick and the first requests results in a 500... so we consider the retries also for 500.